lms/djangoapps/teams/serializers.py

```diff
@@ -58,9 +58,10 @@ class Meta(object):
             "description",
             "country",
             "language",
+            "last_activity_at",
             "membership",
         )
-        read_only_fields = ("course_id", "date_created", "discussion_topic_id")
+        read_only_fields = ("course_id", "date_created", "discussion_topic_id", "last_activity_at")
```

Comment:
Should last_activity_at be read-only? We will be updating it on the server side certainly, but I suppose we don't want to expose the ability through the REST API?